docs/source/general/freeradius.rst

```diff
@@ -239,7 +242,7 @@ and ``Calling-Station-ID``:
     echo $request | radclient localhost auth testing123
```

Comment:
radclient for acct and radtest for auth. 
In the comment we read about radtest but example show us a radclient command

Reply:
Thy are both shown with their relative command